src/main/resources/application.properties

```diff
@@ -1,13 +1,17 @@
+#Sovereign cloud configuration
+authorityEndpointHost="https://login.microsoftonline.com/"
+msGraphEndpointHost="https://graph.microsoft.com/"
+
+#AAD configuration
 aad.clientId=Enter_the_Application_Id_here
 aad.authority=https://login.microsoftonline.com/Enter_the_Tenant_Info_Here/
```

Comment:
@sangonzal This is the value that will set by portal. So it should be fine as it is if the apps.json mentions the value to be replaced is https://login.microsoftonline.com/.
I think what you are missing is having the Graph endpoint in the config and using that in the application code.